### PR TITLE
Add profile info to Welcome page

### PR DESCRIPTION
**Pull request by BriocheBerlin**

### CODE REVIEW

File: app/assets/stylesheets/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    .changeonhover:hover {
+        background-color: blue;
+        width: 600px;
+        height: 300px;
+    }
+</style>
+</head>
+<body>
+<div>Hover me! Hover Me!</div>
+</body>
```
Comment (til) [inline app/assets/stylesheets/index.html]: Was this file added on purpose?

**Comment by til** [comment]
I restarted the travis build, looks like travis had some hiccups.

File: Gemfile.lock [hunk 2]
```diff
@@ -342,6 +342,3 @@ DEPENDENCIES
   turbolinks
   uglifier (>= 1.3.0)
   will_paginate
-
-BUNDLED WITH
```
Comment (berlintam) [inline Gemfile.lock]: oops, sorry @BriocheBerlin I told u to commit this but you shouldn't have actually. @PragTob what do we do now?

Comment (til) [inline Gemfile.lock]: Just revert this change to this file, and push that as new commit to this branch?

Bonus points for squashing it, but can be done later.

Oh, and run a `gem install bundler` locally to make sure you have the most recent one installed. Then you should be safe from getting this change applied automatically again, I think there were a few recent versions of bundler that always changed this section, but the most recent one only does it if the Gemfile.lock also changed elsewhere.

File: app/assets/stylesheets/index.html (hunk 1 above)
Comment (PragTob) [inline app/assets/stylesheets/index.html]: I don't think so :)

File: Gemfile.lock (hunk 2 above)
Comment (PragTob) [inline Gemfile.lock]: What @til said :)

I can also do the squashin :)

Comment (berlintam) [inline Gemfile.lock]: thanks @til & @PragTob!

**Pull request merged**